### PR TITLE
fix login error

### PR DESCRIPTION
**Pull request by Yeshasvitvs**

### CODE REVIEW

File: src/userinfo.py [hunk 1]
```diff
@@ -39,7 +39,8 @@ def hello(self):
     def get_user_id_by_login(self, user_name):
         url_info = self.url_user_info % (user_name)
         info = self.s.get(url_info)
-        json_info = json.loads(re.search('{"activity.+show_app', info.text, re.DOTALL).group(0)+'":""}')
+        json_info = json.loads(re.search('{"activity.+probably_has_app', info.text, re.DOTALL).group(0)+'":""}')
+        print json_info
```
Comment (MP0w) [inline src/userinfo.py]: left over!

**Comment by Yeshasvitvs** [comment]
@MP0w cleaned the leftover

**Comment by MP0w** [comment]
@Yeshasvitvs cool, however I'm not a maintainer of this repo. I just checked the code and saw it so suggested to remove it. I have not rights to merge the PR.

**Comment by ericwoolard** [comment]
Looks like it's been changed again - https://github.com/instabot-py/instabot.py/issues/1522#issuecomment-416006669

**Comment by doskadev** [comment]
+

**Comment by adamevers** [comment]
Works great! Thanks for the PR.

**Pull request closed without merging**